app/components/public/session-item.hbs

```diff
@@ -83,8 +83,7 @@
     {{/if}}
     
     {{#each @session.speakers as |speaker|}}
-      <div class="ui divider"></div>
```

Comment:
what if there are more than 1 speaker? Without a divider. How it looks? share a screenshot.

Reply:
![withoutDiv](https://user-images.githubusercontent.com/64387054/96902026-a1b9b000-14b1-11eb-91c4-0766cdb3b751.JPG)

it looks like this